fe/fe-core/src/main/java/com/starrocks/qe/ShowExecutor.java

```diff
@@ -666,12 +667,27 @@ private void handleShowCreateTable() throws AnalysisException {
             }
 
             if (table instanceof View) {
+                if (showStmt.isMaterializedView()) {
```

Comment:
```suggestion
                if (!showStmt.isView()) {
```

Reply:
if the statement is 'show create table' and the tableType is MATERIALIZED_VIEW or VIEW, the table can be shown